manifests/site.pp

```diff
@@ -20,24 +38,21 @@
     content => inline_template($host_template)
   }
 
-$config_containerd = @(END)
-disabled_plugins = []
-END
+  $api_server_count = size(lookup('terraform.tag_ip.controller'))
 
-  file { '/etc/containerd':
-    ensure => 'directory',
-    tag    => 'mc_bootstrap',
+  $filteredControllers = $instances.filter |$name, $info| {
+    'controller' in $info['tags']
   }
-
-  file { '/etc/containerd/config.toml':
-    ensure  => file,
-    content => inline_template($config_containerd),
-    require => File['/etc/containerd'],
-    tag    => 'mc_bootstrap',
+  $controllersIps = $filteredControllers.map |$name, $info| {
+    "${name}:${info['local_ip']}"
```

Comment:
The right. form for etcd_initial_cluster is described here:
https://github.com/puppetlabs/puppetlabs-kubernetes/blob/main/manifests/init.pp#L210

Reply:
This part is not present anymore. It could be useful if we generate the certificate our-self, but for now, kubetool generate it and it's merged in the hiera data